decentraland/unity-renderer
# re-revert: uishape polling, with extra fix

## What does this PR change?

This PR re-reverts #2756

But also contains #2788 once its merged

# How to test

Go to [ICE Poker](https://play.decentraland.org/?renderer-branch=fix/uishape-regression&realm=dg&position=-100%2C126) be sure to connect with a wallet.

The loading of the scene should be as smooth as it is in production, when everything is loaded finally, it should be faster ( a bit )

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
